src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+// import books from '../../components/BookItem';
```

Comment:
- [ ] I kindly suggest you remove commented out code from your codebase as it doesn't make it look neat.